frontend/src/Frontend/UI/Dialogs/AddAccount.hs

```diff
@@ -107,15 +125,21 @@ uiCreateWalletStepOne model onClose = Workflow $ do
           pure (net, chain, mkAccountNotes notes)
         newConf = mempty & walletCfg_createWalletOnlyAccount .~ eAddAcc
 
+        eInflightFound = () <$ ffilter isNothing (updated dInflightAcc)
```

Comment:
I'd expect `isJust` ?

Reply:
I want this to fire only when when an inflight account has been 'found' on the chain, and therefore no-longer inflight so we have to jimmy up a new pair of keys.

So that would change the value in the dynamic from `Just account` to `Nothing`, so only fire on `Nothing`. I have to look up which direction the `filter` function is using the predicate *EVERY SINGLE TIME* ... so I wouldn't be surprised if I need to flip it.